source/MaterialXGenShader/Nodes/SourceCodeNode.cpp

```diff
@@ -173,17 +190,22 @@ void SourceCodeNode::emitFunctionCall(const ShaderNode& node, GenContext& contex
         else
         {
             // An ordinary source code function call
+            string delim = "";
```

Comment:
Although this was already present in the original code, I'll note that the `= ""` is not needed here, as this matches the behavior of the default string constructor.